### PR TITLE
Restaurant Page - Use of Package Manager

### PR DESCRIPTION
**Pull request by happiguru**
This PR is a simple Restaurant website that implements the following features:

1. Uses ES6 Concept for modern Web applications
2. Uses the Webpack Package Manager to manage files
3. Implements the use of import and export statements to functions from modules
4. more
Thank you

### CODE REVIEW

File: src/reservation.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  const artsection = document.querySelector('.caption-title');
+  const art = document.createElement('article');
+  const heading1 = document.createElement('h1');
+  const firstpara = document.createElement('p');
+  const formm = document.createElement('form');
+  const input1 = document.createElement('input');
+  const input2 = document.createElement('input');
+  const input3 = document.createElement('input');
+
+  // secondpara.classList.add("contactpage");
```
Comment (christianotieno) [inline src/reservation.js:14]: ## Remove comments
- [x] Comments: It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

**Review by christianotieno — CHANGES_REQUESTED** [review]
## Changes Required ♻️ 
Hello @happiguru 👋 
Everything looks great, Let's take care of a minor issue first before we merge this. 👇 

### Happy coding 💻 
### Christian Otieno(TSE)

File: src/reservation.js (hunk 1 above)
Comment (happiguru) [reply src/reservation.js:14]: Removed comment

**Review by addod19 — APPROVED** [review]
## Status: Approved :trophy: :balloon: 

Hi @happiguru, congratulations :partying_face: having met all the minimum requirements, it is my privilege to approve this project :fireworks: Congratulations once again:trophy: 

Kindly go ahead and merge into development :+1: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Pull request merged**